packages/frontend/src/components/common/Footer.js

```diff
@@ -133,7 +133,7 @@ const Footer = () => {
             <div className='right'>
                 <Translate id='footer.needHelp' /><br />
                 <a
-                    href='https://near.chat'
+                    href='https://discord.com/invite/Vj74PpQYsh'
```

Comment:
Should we change this based on the `isWhitelabel` function?

Reply:
Sure! Fixed.